lib/html.php

```diff
@@ -244,16 +244,26 @@ function convertLazyLoading($dom)
         $dom = str_get_html($dom);
     }
 
+    // Retrieve image URL from srcset attribute
+    // https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset
+    // Example: convert "header640.png 640w, header960.png 960w, header1024.png 1024w" to "header1024.png"
+    $srcset_to_src = function ($srcset) {
+        $sources = explode(',', $srcset);
+        $last_entry = trim(end($sources));
```

Comment:
`array_pop`

Reply:
Thanks for reviewing my pull request.
Would you mind to elaborate why `array_pop` should be preferred over `end`? I'm curious.

Reply:
the purpose of `reset()` and its cousin `end()` is to modify internal pointer of an array. their original purpose is not to get the first/last value of an array.

it's just more readable to use `array_pop()` or `array_last()` (if it ever comes into existance).

also acceptable: `$last_entry = $sources[array_key_last($sources)];`

Reply:
Okay, thanks. I'll update this in my next pull request.